glossary.yml

```diff
@@ -1477,6 +1477,13 @@
     def: >
       የ [R](#r_language) ቋንቋን የመሠረቱ መሠረታዊ ተግባራት። የመሠረት እሽጎች በ ውስጥ ሊገኙ ይችላሉ "src / library" እና ከ R ውጭ አይዘመኑም; የእነሱ ስሪት ቁጥሮች አር የስሪት ቁጥር። የመሠረት ፓኬጆች ተጭነው በ R ተጭነዋል ፣ ቅድሚያ የሚሰጠው ግን ፓኬጆች ከመሠረታዊ አር ጋር ተጭነዋል ነገር ግን ከመጠቀምዎ በፊት መጫን አለባቸው ፡፡ የ [R](#r_language) ቋንቋን የመሠረቱ መሠረታዊ ተግባራት። የመሠረት እሽጎች በ "src/library" ውስጥ ሊገኙ ይችላሉ እና ከ R ውጭ አይሻሻሉም; የእነሱ ስሪት ቁጥሮች የአር  ስሪት ቁጥርን ይከተላሉ።  መሠረት ፓኬጆች ጋር ከR ተጭነዉ ይዘረጋሉ፣ ቅድሚያ የሚሰጠው ግን ፓኬጆች ከመሠረታዊ አር ጋር ተጭነዋል ነገር ግን ከመጠቀም በፊት መጫን አለባቸው ፡፡
 
+  ar:
+    term: "آر الأساس"
+    def: >
+     `src/library` يمكن الحصول علي الحزم الأساسية في .[R](#r_language) الوظائف الأساسية التي تتكون منها لغة آر
+     R الحزم الأساسية مثبتة ويتم تحميلها مع R أرقام نسختها تتبع ترقيم نسخة R لايمكن تحديث الحزم الأساسية خارج
+ولكن يجب أن يتم تحميل قبل استخدامها .  R بينما الحزم ذات الألوية يتم تثبيتها مع 
+
```

Comment:
I _think_ this might help, but I'm not confident with RTL languages and YAML 

```suggestion
      `src/library` يمكن الحصول علي الحزم الأساسية في .[R](#r_language) الوظائف الأساسية التي تتكون منها لغة آر
      R الحزم الأساسية مثبتة ويتم تحميلها مع R أرقام نسختها تتبع ترقيم نسخة R لايمكن تحديث الحزم الأساسية خارج
ولكن يجب أن يتم تحميل قبل استخدامها .  R بينما الحزم ذات الألوية يتم تثبيتها مع 

```

Reply:
@carpentries/glosario-editors-ar, can you help?